packages/elements/src/tree/managers/tree-manager.ts

```diff
@@ -437,6 +438,8 @@ export class TreeManager<T extends TreeDataItem> {
    * @returns {void}
    */
   public uncheckAllItems (): void {
-    this.editableItems.forEach(item => this.uncheckItem(item));
+    // uncheck items from top levels when manage relationships to avoid redundant re-renders
+    const items = this.manageRelationships ? this.composer.queryItems(() => true, 0) : this.checkedItems;
```

Comment:
I'm not sure about difference items to action  will cause the problem for checkAllItems and uncheckAllItems?
For example: 
uncheckAllItems will force uncheck even if item is lock
while checkAllItems can check only editable item

Reply:
It will not uncheck lock items because inside uncheckItem method has condition to check is item checkable.